proto/ibc/core/channel/v2/packet.proto

```diff
@@ -14,9 +14,9 @@ message Packet {
   // with a later sequence number.
   uint64 sequence = 1;
   // identifies the sending chain.
-  string source_channel = 2;
+  string source_id = 2;
```

Comment:
It will be the last time I bring this up (I hope), but what is the reason we don't want to name source and destination identifier client_id (as in source_client_id)? Can it be anything else?